bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -516,64 +516,64 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
                            (if it will work on an exact match, leave use_init_interp off)
 -->
 
-<finidat hgrid="0.9x1.25"  maxpft="17"  mask="gx1v6" use_cn=".true." use_cndv=".false." use_fates=".false."
+<finidat hgrid="0.9x1.25"  maxpft="17"  mask="gx1v7" use_cn=".true." use_cndv=".false." use_fates=".false."
          ic_ymd="18500101" use_nitrif_denitrif=".true." use_vertsoilc=".true." sim_year="1850"
-         ic_tod="0" glc_nec="10" use_crop=".false." irrigate=".false." use_init_interp=".true."
```

Comment:
This looks right to me. Since, the files now don't require interpolation, you leave off use_init_interp=T for each file, and let it only set it if it isn't an exact match. This section here says you can do interpolation if CNDV is off, FATES is off, and the sim year is 1850 or 2000. In other cases you can't do interpolation.